src/components/Layout.js

```diff
@@ -78,9 +78,12 @@ const Layout = (props) => {
       setShouldShowSideNav(true)
 
       if (localStorage.getItem("zen-mode") !== null) {
-        const isMobile = /Android|webOS|iPhone|iPad|iPod|BlackBerry|IEMobile|Opera Mini/i.test(
-          navigator.userAgent
-        )
+        let isMobile = false
+        if (typeof window !== undefined) {
```

Comment:
> Did a little cleanup, but looks good @Hakeemmidan! Thanks!

@corwintines I think this needs to be 
```JavaScript
if (typeof window !== 'undefined') {
```
Because [the docs say](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/typeof): "The **typeof operator returns a string** indicating the type of the unevaluated operand."

Other than that, the new changes look good! Thank you!